pyproject.toml

```diff
@@ -143,6 +143,8 @@ dependencies = [
   "httpx", # For OpenAI testing
   "respx", # For OpenAI testing
   "nest-asyncio", # for executor testing
+  "astunparse; python_version<'3.9'",
```

Comment:
A comment here would be helpful so we know why this one is necessary for 3.8

Reply:
ok